### PR TITLE
Update LLVM 4.0 & added pictures

### PR DESCRIPTION
**Pull request by enovella**

### CODE REVIEW

File: quickstart/README.md [hunk 1]
```diff
@@ -69,10 +69,19 @@ Fuzzing
 
 Fuzz it:
 
-    $ ~/afl-2.45b/afl-fuzz -i inputs -o out ./vulnerable
+    $ ~/afl-2.45b/afl-fuzz -i in -o out ./vulnerable
```
Comment (mykter) [inline quickstart/README.md:72]: the folder is called inputs in this case

File: quickstart/README.md [hunk 2]
```diff
@@ -69,10 +69,19 @@ Fuzzing
...
 For comparison you could also test without the provided example inputs, e.g.:
 
     $ mkdir in
     $ echo "my seed" > in/a
     $ ~/afl-2.45b/afl-fuzz -i in -o out ./vulnerable
+
+POC
+====
+
+Without instrumentation (`-n`): (`afl-fuzz -i inputs -o out -n ./vulnerable`)
```
Comment (mykter) [inline quickstart/README.md:83]: Whilst it's an interesting comparison, I don't think I want to go into the differences between these two modes in the quickstart - we just want to have the bare minimum to get started fuzzing.

File: quickstart/README.md [hunk 3]
```diff
@@ -69,10 +69,19 @@ Fuzzing
...
+![imaing](https://github.com/enovella/afl-training/blob/master/quickstart/pics/afl-pic-vulnerable-dumbf.png)
+
+With instrumentation (`afl-clang-fast`):
+![imaing](https://github.com/enovella/afl-training/blob/master/quickstart/pics/afl-pic-vulnerable.png)
```
Comment (mykter) [inline quickstart/README.md:87]: A screenshot is a nice idea, thanks. I've added one to master that shows the status once all three easily findable crashes have been detected.

**Review by mykter — CHANGES_REQUESTED** [review]
Thanks for the PR! Glad to see it works with clang 4.0, happy to merge in that part of the request.
I've left some comments against the other parts.

I hope you found it useful. How did you come across it, out of interest?

**Comment by andronat** [comment]
Hey @enovella! Any news on this?

**Comment by enovella** [comment]
Sorry but I am extremely busy at the moment with other things.

**Comment by mykter** [comment]
I've updated the Dockerfile to LLVM 6, so closing this now.

**Pull request closed without merging**